root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -953,22 +953,22 @@ const {LINK_TYPES, cleanURL, guessType, validationRules} = require('../../edit/U
         },
         // Dailymotion
         {
-                             input_url: 'https://dailymotion.com/who-knows#uploads',
+                             input_url: 'http://dailymotion.com/who-knows#uploads',
```

Comment:
There is no more test with `https` input Dailymotion URL.

Reply:
Should there be? It's not like https is going to get cleaned up to anything but https. But I can change one back if desired :)

Reply:
Just making sure the it is still/will still be recognized by the system. This kind of regression happened in the past.

Reply:
Ok, changed one for this :)